apps/comps/rpc/router/boost/claim-staked-boost.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
```

Comment:
Implementation of `claimStakedBoost`, that calls `claimStakedBoost` from the service.